x-pack/plugins/monitoring/server/lib/beats/types.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ElasticsearchResponse } from '../../../common/types/es';
```

Comment:
This seems fine, but it's a little curious. Is there a reason we import ElasticsearchResponse via beats/types rather that importing from common/types/es directly?